pkg/kubelet/server.go

```diff
@@ -260,12 +260,12 @@ func (s *Server) handleContainerLogs(w http.ResponseWriter, req *http.Request) {
 
 // handleBoundPods returns a list of pod bound to the Kubelet and their spec
 func (s *Server) handleBoundPods(w http.ResponseWriter, req *http.Request) {
-	pods, err := s.host.GetBoundPods()
+	pods, err := s.host.GetPods()
 	if err != nil {
 		s.error(w, err)
 		return
 	}
-	boundPods := &api.BoundPods{
```

Comment:
I can't see any client using this handler. But I only did it by code "grep" and I'm not sure how should I check it in more legitimate way...